x-pack/legacy/plugins/siem/public/components/url_state/__snapshots__/index.test.tsx.snap

```diff
@@ -1,547 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
The contents of this snapshot file are now empty. Is that expected?

Reply:
yes!!!!